### ISSUE TITLE
Technology changes

### ISSUE THREAD

**Issue by AllanJeremy** [AllanJeremy/esomo-upgrade#38]
@mwauramuchiri  As we are both aware, we will need to rewrite the system in a framework. Codeigniter would be the first choice for us since we are both familiar with it.

However, Laravel seems to be more practical as far as features are concerned, it also has websocket support and can easily intergrate with vue/react. However it is complicated.

What are your thoughts

**Comment by mwauramuchiri** [comment]
True Laravel would take longer to move to first. I suggest we do everything to consider the platform complete first then test it in one or more schools.
How long do you think we could move to Codeigniter? I think a week. At most two, since we are ninjas at the framework... hehe.
That means maybe by the end of the third quarter we should be moving to Laravel slowly.

**Comment by AllanJeremy** [comment]
Agreed, hadn't thought of that. Yeah it would probably take a week or two at most. Hehe, yes we are. 

Then move to laravel once we have it working in CI. Oh and in the meantime, could you disable the comment feature for now? Like hiding the buttons to allow for chats. We can add this in the next update as ajax and setInterval before we change tech and have websockets. We need to work on **assignments section for students**. and the **view assignment submissions**. Other than that, test timer is working, only getting and displaying test results is left (working on this today). 

Let's try and close two schools in this quarter ama?

**Comment by mwauramuchiri** [comment]
Yes two schools! Just the thought of it is exciting already!

I'll disable the comment buttons today and have different proposals of the **view assignment submissions** that we'll discuss

**Comment by AllanJeremy** [comment]
Alright perfect. Will make phone calls and get some meetings to pitch to schools this week.

Hangouts tonight for that?
